File: back-brisofus/src/main/java/com/baudec/brisofus/entity/ItemFilter.java
package com.baudec.brisofus.entity;

import lombok.*;

import java.util.List;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@ToString
public class ItemFilter {

    public String searchName;
    public int levelMin;
    public int levelMax;
    public List<String> listAttribute;
    public List<String> listType;


}

File: BrisofusFront/src/app/models/ItemFilter.ts
export default class ItemFilter{
    listAttribute:String[];
    listType:String[];
    levelMin:number;
    levelMax:number;
    searchName:String;

    constructor(){
        this.listAttribute=[];
        this.listType=[];
        this.levelMax=60;
        this.levelMin=40;
        this.searchName=""
    }

    removeListAttribute(attribute:String){
        this.listAttribute = this.listAttribute.filter(obj => obj !== attribute);
    }
}